internal/cmd/org.go

```diff
@@ -385,6 +388,72 @@ var membersInviteCmd = &cobra.Command{
 	},
 }
 
+var inviteRemoveCmd = &cobra.Command{
+	Use:               "invite-remove <email>",
+	Short:             "Remove a pending invite to an email to join the current organization",
+	Args:              cobra.ExactArgs(1),
+	ValidArgsFunction: noFilesArg,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
+		settings, err := settings.ReadSettings()
+		if err != nil {
+			return err
+		}
+
+		email := args[0]
+		if email == "" {
+			return fmt.Errorf("email cannot be empty")
+		}
+
+		client, err := authedTursoClient()
+		if err != nil {
+			return err
+		}
+
+		if err := client.Organizations.DeleteInvite(email); err != nil {
+			return err
+		}
+
+		org := settings.Organization()
```

Comment:
IIRC this can be empty for personal organizations. Please verify that if you haven't already!